www/layouts/educator/section.html

```diff
@@ -116,8 +117,7 @@ <h1 class="container font-cardo font-weight-normal quote col-md-8 offset-md-2">
                 </h1>
                 <p class="font-Helvetica author font-weight-normal">- Professor Jonathan Gruber</p>
                 <h3 class="font-cardo chalk-radio font-weight-normal mt-5">
-                  <b>Chalk Radio</b> &nbsp;Season 1 Eposide 9 Thinkin Like an
-                  &ldquo;Economist&rdquo;
+                  <b>Chalk Radio</b> &nbsp;Season 1 Eposide 9 Thinking Like an &ldquo;Economist&rdquo;
```

Comment:
this line has another typo in it: "Eposide" should be "Episode", and the 'Thinking' part should be like `&ldquo;Thinking like an Economist&rdquo;`

```suggestion
                  <b>Chalk Radio</b> &nbsp;Season 1 Episode 9 &ldquo;Thinking Like an Economist&rdquo;
```